srml/slashing/src/tests.rs

```diff
@@ -0,0 +1,161 @@
+#![cfg(test)]
+
+use super::*;
+use runtime_io::with_externalities;
+use srml_support::{decl_module, impl_outer_origin, dispatch::Result};
+use system::{self, ensure_signed};
+use substrate_primitives::{Blake2Hasher, H256};
+use primitives::{
+	BuildStorage,
+	traits::{BlakeTwo256, IdentityLookup},
+	testing::{Digest, DigestItem, Header}
+};
+
+impl_outer_origin! {
+	pub enum Origin for Test {}
+}
+
+type Balance = balances::Module<Test>;
```

Comment:
you could just make things a lot easier for yourself and alias `Balance` to `u64` in tests if you'd like.